packages/components/header/src/header.css

```diff
@@ -24,23 +24,26 @@
 	}
 }
 
-::slotted(:first-of-type) {
-	margin-left: auto;
+.left {
+	align-items: center;
+	display: flex;
 }
 
-::slotted(*) {
-	margin-right: var(--ts-unit-half);
+.right {
```

Comment:
right and left class names are a bit confusing when it comes to the `rtl` direction, but not super important if no clear alternative.

Reply:
Good point. What do you think about `start` and `end`, or `title-container` and `slot-container`?

Reply:
title-icon-container(wrapper`another convention to discuss I guess` :thinking:) seems a better option to me 👍